packages/labs/testing/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "tsBuildInfoFile": ".tsbuildinfo",
     "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true,
-    "types": ["mocha"]
+    "types": ["mocha"],
+    "skipLibCheck": true
```

Comment:
was this addition necessary?

Reply:
Yes, as the chokidar types fail with the Node.js types update.